core/cap-0023.md

```diff
@@ -239,146 +293,248 @@ default:
-edge case, it does simplify the behavior of `BalanceEntry` by guaranteeing that
-the receiving account always receives both the reserve and the amount of asset.
-
-### BalanceEntry createdBy and claimableBy are accountIDs
-Each `BalanceEntry` can only be claimed by `claimableBy` or `createdBy` (exactly
-one of them depending on the ledger close time). As a consequence, it is possible
-for a `BalanceEntry` to become stranded. If `claimableBy` is merged and the
-private key is not known, then the `BalanceEntry` will never be claimable by the
-intended recipient. Similarly if `createdBy` is merged and the private key is not
-known, then the `BalanceEntry` will never be reclaimable by the sender.
-
-Suppose that we try to relax this requirement in order to avoid this downside. We
-could instead attach two public keys to the `BalanceEntry`, where the sender alone
-knows one of the private keys and the recipient alone knows the other. Which key is
-currently usable would be determined by the ledger close time. The operation to
-claim the `BalanceEntry` could then specify an account A and a signature over A
-from the key that is currently usable. This would allow the appropriate party to
-claim the `BalanceEntry` into any account that they control. But this would also
+   been claimed
+3. Receiving accounts always receive the reserve from the claimed
+   `ClaimableBalanceEntry`
+
+The third benefit facilitates a desirable optimization in which a
+`ClaimableBalanceEntry` can be directly converted into a `TrustLine`. This is
+useful in some asset issuance workflows where the issuer wants to pay for the
+trust line but does not want the recipient to spend the funds prior to creating
+the trust line. Of course if the base reserve were raised after the
+`ClaimableBalanceEntry` was created then the reserve received would not be
+sufficient to pay for the trust line, but they would at least still receive the
+reserve. But if the `ClaimableBalanceEntry` were a sub-entry of the sending
+account and the base reserve were raised such that the sending account no longer
+satisfies the reserve requirement then the receiving account would receive no
+reserve (because accounts that do not satisfy the reserve requirement cannot
+have their native balance decrease). While this is an unlikely edge case, it
+does simplify the behavior of `ClaimableBalanceEntry` by guaranteeing that the
+receiving account always receives both the reserve and the amount of asset.
+
+### ClaimableBalanceEntry claimants are accounts
+For each `ClaimableBalanceEntry`, `condition` contains a finite and immutable
+list of accounts that could potentially claim the `ClaimableBalanceEntry`. Even
+if the conditions are satisfiable (which is not guaranteed), it is still
+possible for the `ClaimableBalanceEntry` to become stranded. If all of the
+accounts listed in `condition` are merged and none of the private keys are
+known, then the `ClaimableBalanceEntry` will no longer be claimable.
```

Comment:
Dangling claimables seems problematic. This seems like something that could be footgun.